### PR TITLE
Rename architecture for _zt specs

### PR DESCRIPTION
**Pull request by vsebe**
Rename architecture to arch.znext for _zt specs.

Signed-off-by: Violeta Sebe <vsebe@ca.ibm.com>

### CODE REVIEW

**Comment by vsebe** [comment]
@llxia @renfeiw please review, thank you!

File: resources/ottawa.csv [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
...andard,arch:32bit,os:aix,os:unix,arch:ppc,test:java9","vm:standard,arch:64bit,os:aix,os:unix,arch:ppc,test:java9","vm:compressed_refs,arch:64bit,os:aix,os:unix,arch:ppc,test:java9",,"vm:standard,arch:31bit,os:zos,arch:390,test:java9","vm:standard,arch:31bit,os:zos,arch:znext,test:java9","vm:standard,arch:64bit,os:zos,arch:390,test:java9","vm:compressed_refs,arch:64bit,os:zos,arch:390,test:java9","vm:compressed_refs,arch:64bit,os:zos,arch:znext,test:java9","vm:mixed_refs,arch:64bit,os:zos,arch:390,test:java9","vm:mixed_refs,arch:64bit,os:zos,arch:znext,test:java9","arch:32bit,os:linux,arch:arm","vm:standard,arch:64bit,os:osx,os:unix,arch:x86,test:java9","vm:compressed_refs,arch:64bit,os:osx,os:unix,arch:x86,test:java9",,"vm:standard,arch:64bit,os:osx,os:unix,arch:aarch64,test:java9",,,,"vm:standard,arch:64bit,os:linux,os:unix,arch:aarch64,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:aarch64,test:java9",,"vm:standard,arch:64bit,os:linux,os:unix,arch:riscv,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:riscv,test:java9","vm:mixed_refs,arch:64bit,os:linux,os:unix,arch:riscv,test:java9","vm:mixed_refs,arch:64bit,os:linux,os:unix,arch:x86,test:java9"
```
Comment (llxia) [inline resources/ottawa.csv:4]: Sorry, it is hard for me to see. What was changed here?

Comment (vsebe) [reply resources/ottawa.csv:4]: Renamed `arch:zartemis` to `arch:znext`.

**Review by renfeiw — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]

**Pull request merged**